src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: #{$card-width};
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    display: block;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-style: normal;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__product-rating {
+    margin-bottom: 24px;
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__product-cost {
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__product-price {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__product-price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__product-buy {
+    height: 40px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #00acdc;
+    border: 1px solid #00acdc00;
```

```suggestion
    border: 1px solid;
```
You don't need it, this border will be transparent by default